maps/map_files/DesertDam/standalone/crashlanding-upp-alt1.dmm

```diff
@@ -836,8 +832,7 @@
 /area/desert_dam/exterior/valley/valley_civilian)
 "Ny" = (
 /obj/structure/closet/crate/ammo,
-/obj/item/ammo_magazine/pkp,
-/obj/item/weapon/gun/pkp,
+/obj/item/weapon/gun/rifle/type71/flamer,
```

Comment:
dont forget the ammo for the type 71 you are giving.

Reply:
I think it being loaded with AP is fine since I reduced overall AP mags to compensate.